app/services/driver_pool.py

```
@@ -0,0 +1,43 @@
+    def _create_driver(self):
+        """새로운 WebDriver 인스턴스를 생성합니다."""
+        chrome_options = Options()
+        chrome_options.add_argument("--headless=new")
+        chrome_options.add_argument("--no-sandbox")
+        chrome_options.add_argument("--disable-gpu")
+        chrome_options.add_argument("--window-size=1280x1696")
+        chrome_options.add_argument("--single-process")
+        chrome_options.add_argument("--disable-dev-shm-usage")
+        chrome_options.add_argument("--disable-dev-tools")
+        chrome_options.add_argument("--no-zygote")
+        chrome_options.add_argument(f"--user-data-dir=/tmp/user-data")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)